website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+
+The provisioned instances must comparable permissions to a standard EKS self-managed node. For example, permission to configure instance networking interfaces. The getting started guide again provides a useful starting point in [the cloudformation template](../getting-started/cloudformation.yaml).
+
+### Tagged Subnets
+
+Karpenter needs to discover VPC (virtual private cloud) subnets on AWS, such that requests for new instances are places on the proper subnet, enabling communication with the control plane. Karpenter discovers subnets using resource tags. 
+
+The tags must have this form:
+- key: `kubernetes.io/cluster/${CLUSTER_NAME}`
+- value: "" (empty string)
+
+In some situations, EKS may automatically apply this tag to subnets. Do not rely on EKS to do this. 
+
+### `eksctl`
+
+The getting started guide uses `eksctl`. This tool is officially supported and sponsored by AWS. However, equivalent EKS clusters may be created using CloudFormation, or even the web console. 
+
+The ["cluster.yaml" file](../getting-started/#create-a-cluster) provides does succinctly describe a suitable EKS cluster for Karpenter, including an OIDC provider for IAM (the infrastructure for IRSA). 
+
+`eksctl` also handles interconnecting Kubernetes RBAC and AWS IAM. For example, associating the `karpenter` service account with an IAM role for creating instances. 
+
+```
+eksctl create iamserviceaccount \
+  --cluster $CLUSTER_NAME --name karpenter --namespace karpenter \
+  --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/KarpenterControllerPolicy-$CLUSTER_NAME \
+  --approve
+```
+
+```
+eksctl create iamidentitymapping \
+  --username system:node:{{EC2PrivateDNSName}} \
+  --cluster  ${CLUSTER_NAME} \
+  --arn arn:aws:iam::${AWS_ACCOUNT_ID}:role/KarpenterNodeRole-${CLUSTER_NAME} \
+  --group system:bootstrappers \
+  --group system:nodes
+```
+
+
+### Launch Template
+
+A launch template is an AWS resource that bundles every configuration value needed to properly launch a new instance, such as base image and storage.
+
+Karpenter will create a launch template suitable for the environment described in the getting started guide (EKS, IRSA, etc) if none is specified.
+
+To the extent possible, nodes should be customized using Karpenter provisioner CRD fields. However, many environments require using a different base image (instead of the EKS optimized flavor of Amazon Linux 2), such as to incorporate a standardized security daemon at the node level. 
+
+Review the [detailed instructions](../cloud-providers/aws/launch-templates/) for creating a custom launch template. 
+
+## Deploy Karpenter
+
+The yaml files comprising the helm chart are suitable for production environments. They may be lightly modified for direct use, for example storage in an internal GitOps cluster repo (without referencing an external chart).

```

Comment:
>They may be lightly modified for direct use, for example storage in an internal GitOps cluster repo (without referencing an external chart).

This was confusing for me and the call-out to GitOps seems out of place IMO.

Reply:
good point. revised.